sklearn/preprocessing/data.py

```diff
@@ -1767,23 +1770,25 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
         Will return sparse matrix if set True else will return an array.
 
     handle_unknown : str, 'error' or 'ignore'
-        Whether to raise an error or ignore if a unknown categorical feature is
-        present during transform.
+
+        - 'ignore': Ignore all unknown feature values.
+        - 'error': Raise an error when the value of a feature is unseen during
+          `fit` and out of range of values seen during `fit`.
+        - 'error-strict': Raise an error when the value of a feature is unseen
+          during`fit`.
+
+    copy : bool, default=True
+        If unset, `X` maybe modified in space.
 
     Attributes
     ----------
-    active_features_ : array
-        Indices for active features, meaning values that actually occur
-        in the training set. Only available when n_values is ``'auto'``.
+    feature_index_range_ : array, shape [n_feature, 2]
```

Comment:
The incumbent `feature_indices` does not take non-categorical features into account. I chose start and end indices because the features are not encoded in the same sequence as seen in the input (the non-categorical features and appended at the end of the one-hot encoded array). If the `categorical_features` mask enforces of alternating categorical/non-categorical variables, the range of indices occupied by each feature in the input array will not be sequential.

Here is a snippet to illustrate
```python
from sklearn.preprocessing import OneHotEncoder
import numpy as np

enc = OneHotEncoder(categorical_features=[True, False, True])
X = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
print(enc.fit_transform(X).toarray())
print(enc.feature_index_range_)
```
Outputs
```text
[[ 1.  0.  0.  1.  0.  0.  2.]
 [ 0.  1.  0.  0.  1.  0.  5.]
 [ 0.  0.  1.  0.  0.  1.  8.]]
[[0 3]
 [6 7]
 [3 6]]
```
 I hope I made sense